dev/env/opensource/development.env

```diff
@@ -2,3 +2,19 @@ APP_MODE=full
 APP_ENV=development
 APP_INSTALLATION=opensource
 APP_DEV_MODE=1
+
+HC=1
```

Comment:
We don't enable HC by default

Reply:
but this is for development, without setting this param I get something like this (see screenshot) and have no idea where to search for solution ) and 5 min later I debug handler of the button and in internals fish out something like "Error: Unknown visualization at..." which barely gives me a clue 
<img width="944" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/151746694/65a3ac6e-d776-42c7-b057-90862be6d1c6">

Reply:
Yep, but this is important for legal reasons. You can create you own `.env`